### ISSUE TITLE
1.3.2 and 1.3.3 are not idempotent (conflicting check/result)

### ISSUE THREAD

**Issue by Thulium-Drake** [ansible-lockdown/RHEL8-CIS#93]
The result of the first task (1.3.2) seems to break the check for the second task (1.3.3.)

Everytime I run this role against my system (it has run before on this system) it will change there parameters around.

```
TASK [cis_hardening_el8 : SCORED | 1.3.2 | PATCH | Ensure sudo commands use pty] *************************************
--- before: /etc/sudoers (content)
+++ after: /etc/sudoers (content)
@@ -85,7 +85,7 @@
 #
 # Defaults   env_keep += "HOME"
 
-Defaults    logfile="/var/log/sudo.log"
+Defaults    use_pty
 
 ## Next comes the main part: which users can run what software on 
 ## which machines (the sudoers file can be shared between multiple

changed: [ansible.lbhr.htm.lan]

TASK [cis_hardening_el8 : SCORED | 1.3.3 | PATCH | Ensure sudo log file exists] **************************************
--- before: /etc/sudoers (content)
+++ after: /etc/sudoers (content)
@@ -85,7 +85,7 @@
 #
 # Defaults   env_keep += "HOME"
 
-Defaults    use_pty
+Defaults    logfile="/var/log/sudo.log"
```

**Comment by Thulium-Drake** [comment]
It seems this was fixed in ```april_2021_fixes```

When is the next release of the role planned? :-)

**Comment by georgenalen** [comment]
Thulium-Drake,
 I'm hoping to get the April_2021_fixes at least merged into devel next week. I was waiting to see if any additional issues came up. The last one you raised is what I'm looking at now and will add into the April_2021_fixes before merging into devel/main. Thanks again for raising the issues!

George

**Comment by Thulium-Drake** [comment]
I'm currently working on a PR for #94  :-)

**Comment by uk-bolly** [comment]
@Thulium-Drake 

Just checking that this issue is now resolved for you?

thanks

uk-bolly

**Comment by Thulium-Drake** [comment]
@uk-bolly 

Yes indeed, fixed as of latest ```april_fixes``` :-)

**Comment by uk-bolly** [comment]
Excellent thanks for raising and confirming.
